modin/test/test_utils.py

```diff
@@ -287,3 +290,48 @@ def test_warns_that_defaulting_to_pandas():
 
     with warns_that_defaulting_to_pandas():
         ErrorMessage.default_to_pandas(message="Function name")
+
+
+def test_assert_dtypes_equal():
+    """Verify that `assert_dtypes_equal` from test utils works correctly (raises an error when it has to)."""
+    from modin.pandas.test.utils import assert_dtypes_equal
```

Comment:
Why is this import not at the header?

Reply:
Because this test was made for this function specifically, don't want to polute the module's namespace with redundant functions

Reply:
We would only import `assert_dtypes_equal`. Which redundant functions are you talking about? Maybe there is some import error when doing this at the header?